src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -575,12 +575,22 @@ public void setUseBulkCopyForBatchInsert(boolean useBulkCopyForBatchInsert) {
     boolean userSetTNIR = true;
 
     private boolean sendTimeAsDatetime = SQLServerDriverBooleanProperty.SEND_TIME_AS_DATETIME.getDefaultValue();
+    private boolean useFmtOnly = SQLServerDriverBooleanProperty.USE_FMT_ONLY.getDefaultValue();
 
     @Override
     public final boolean getSendTimeAsDatetime() {
         return !isKatmaiOrLater() || sendTimeAsDatetime;
     }
 
+    /**
+     * Returns the useFmtOnly value.
+     * 
+     * @return flag for using FMTONLY for parameter metadata querying.
+     */
+    public final boolean getUseFmtOnly() {
```

Comment:
I didn't add them to the interface because I don't expect people to implement this logic. Not all public APIs from SQLServerConnection are in ISQLServerConnection from what I can tell.